cms/grading/__init__.py

```diff
@@ -448,7 +456,7 @@ def compilation_step(sandbox, commands):
 
 
 def evaluation_step(sandbox, commands,
-                    time_limit=0.0, memory_limit=0,
+                    time_limit=0.0, memory_limit=0, max_processes=1,
```

Comment:
I should point out that I'm happy for the previous behaviour to exist, but if that's the case this should default to none + `sandbox.max_processes` should not be set if it's none.